src/org/opencms/main/CmsShell.java

```diff
@@ -674,6 +675,10 @@ public void execute(Reader reader) {
                     m_out.println(line);
                     continue;
                 }
+                if (line.indexOf(KeyEvent.VK_ESCAPE) != -1) {
```

Comment:
This detects under linux if an escape sequence has been typed.
I don't have a windows to test if this is a sensible change. The output should also be internationalized, if deemed appropriate.